src/components/views/rooms/RoomTile.tsx

```diff
@@ -120,6 +120,12 @@ export default class RoomTile extends React.PureComponent<IProps, IState> {
         return !this.props.isMinimized && this.props.showMessagePreview;
     }
 
+    public componentDidUpdate(prevProps: Readonly<IProps>, prevState: Readonly<IState>) {
+        if (prevProps.showMessagePreview !== this.props.showMessagePreview && this.showMessagePreview) {
```

Comment:
`this.showMessagePreview` seems like it won't clear the message preview on the falling edge?

Reply:
Doesn't need to? The property will return false and thus not render.

Reply:
The re-render was already happening ftr, it just didn't have a preview to render so it ultimately didn't appear to be working at all. All we need to do is make sure there's a preview for it to show.